lib/robots/dor_repo/assembly/accessioning_initiate.rb

```diff
@@ -24,9 +24,8 @@ def initialize_workspace
           object_client.workspace.create(source: assembly_item.path_finder.path_to_object)
         end
 
-        def start_accession_workflow
-          current_version = object_client.version.current
-          workflow_service.create_workflow_by_name(druid, 'accessionWF', version: current_version, lane_id:)
+        def close_version
+          object_client.version.close
```

Comment:
Should we check the return value here? What if this can't be closed?

Reply:
If it can't be closed, then an exception will be raised and the workflow step will fail.

I'm not seeing the loop. This is for the assemblyWF to close a version, which will initiate accessionWF.

Reply:
Got it. I was thinking this was accessioningWF.